release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -578,6 +578,24 @@ If you are receiving pod security violations, see the following resources:
 
 * If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.
 
+[discrete]
+[id="ocp-4-13-oc-mirror-openshift-endpoint"]
+=== The oc-mirror plugin now retrieves graph data container images from an OpenShift API endpoint
+
+The oc-mirror OpenShift CLI (`oc`) plugin now downloads the graph-data tarball from an OpenShift API endpoint instead of downloading the entire graph-data repository from GitHub. Retrieving this data from Red Hat instead of an external vendor is more suitable for users with stringent security and compliance restrictions on external content.
```

Comment:
Is "graph-data" hyphenated or not? It looks like it's used both ways in this PR

Reply:
Good point, thanks. I'm not entirely sure, but I'll check and make it consistent in this PR.